source/_docs/z-wave/installation.markdown

```diff
@@ -157,15 +157,12 @@ On the Raspberry Pi you will need to enable the serial interface in the `raspi-c
 
 #### {% linkable_title Linux (except Hassbian) %}
 
-On Debian Linux platforms there two dependencies you will need to have installed ahead of time (included in `systemd-devel` on Fedora/RHEL systems):
```

Comment:
According to the [spec file](https://src.fedoraproject.org/rpms/systemd/blob/master/f/systemd.spec) is `libudev-dev` still handled by `systemd-devel`.